Source/Android/app/src/main/res/layout/list_item_cheat.xml

```diff
@@ -24,14 +24,14 @@
 
     <CheckBox
         android:id="@+id/checkbox"
-        android:layout_width="48dp"
-        android:layout_height="64dp"
```

Comment:
If I recall correctly, I set it like this to make the touch target larger. I assume it's smaller with the change you've made here? Is there a better way available for making the touch target larger?

Reply:
Ah, that makes sense. I will add it again since I don't have any better solution for this right now.